configs/sample_yamls/serviceops/sample_service_config.yml

```diff
@@ -0,0 +1,172 @@
+#
+########################## types ########################
+#
+types:
```

Comment:
As discussed with @mandarjog I don't think these 'types' are adding any value here. They are 1:1 with constructors and the correspondence is likely to hold in more realistic configurations.